package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@interlay/interbtc-api",
   "version": "1.21.0",
-  "description": "JavaScript library to interact with interBTC",
+  "description": "JavaScript library to interact with interBTC and kBTC",
```

Comment:
Paging @gregdhill , I seem to recall you removed this ("and kBTC") in your other PR?